classes/class-twenty-twenty-one-customize.php

```diff
@@ -81,6 +81,38 @@ public function register( $wp_customize ) {
 				)
 			);
 
+			/**
+			 * Add excerpt or full text selector to customizer
+			 */
+			$wp_customize->add_section(
+				'theme_settings',
+				array(
+					'title'    => esc_html__( 'Theme settings', 'twentytwentyone' ),
+					'priority' => 30,
+				)
+			);
+
+			$wp_customize->add_setting(
+				'display_excerpt_or_fullpost',
+				array(
+					'capability' => 'edit_theme_options',
+					'default'    => 'excerpt',
```

Comment:
For the `sanitize_callback` mentioned by @carolinan above (see https://github.com/WordPress/twentytwentyone/pull/354/files#r501613550), it could be as simple as this:
```suggestion
					'capability' => 'edit_theme_options',
					'default'    => 'excerpt',
					'sanitize_callback' => function( $value ) {
						return 'excerpt' === $value || 'full' === $value ? $value : 'excerpt'
					},
```